packages/cli/src/runAndroid/runAndroid.js

```diff
@@ -132,25 +133,25 @@ function runOnSpecificDevice(
         adbPath
       );
     } else {
-      console.log(`Could not find device with the id: "${args.deviceId}".`);
-      console.log('Choose one of the following:');
-      console.log(devices);
+      logger.info(`Could not find device with the id: "${args.deviceId}".`);
+      logger.info('Choose one of the following:');
+      logger.info(devices);
```

Comment:
please fold it into single `logger.error`

Reply:
fixed